packages/ckeditor5-link/tests/linkcommand.js

```diff
@@ -982,6 +1043,156 @@ describe( 'LinkCommand', () => {
 			expect( command.automaticDecorators ).to.be.an.instanceOf( AutomaticDecorators );
 		} );
 	} );
+
+	describe( 'findChanges()', () => {
```

Comment:
It would be better to test the command execution in more complex scenarios to cover all cases instead of exporting this function and testing separately, especially since we do not check if it is used by the link command.

Reply:
Same for the other exported helper.

Reply:
Removed